FlowCrypt/Controllers/SideMenu/Menu/MyMenuViewController.swift

```diff
@@ -28,7 +28,7 @@ final class MyMenuViewController: ASViewController<ASDisplayNode> {
             .first
             .map(String.init) ?? ""
 
-        let email = dataManager.currentUser()?.email
+        let email = dataManager.email?
```

Comment:
Here also I think email is required, and if it's not there, an error is appropriate.

Reply:
https://github.com/FlowCrypt/flowcrypt-ios/issues/140